guide/adoc/portfile-keywords.adoc

```diff
@@ -219,4 +219,21 @@ license_noconflict openssl
 [source]
 ----
 license_noconflict readline gdbm
+----
+use_xcode::
+Available in 2.6.0 and later. By default, it is assumed on ``macosx`` that ports will not need tools from Xcode.app unless (1) Command Line Tools aren't installed, (2) you are on an old version of Mac OS X that does not support the xcode-select mechanism, or (3) the port uses build.type xcode or is in the xcode PortGroup.
+If a port needs to use Xcode (i.e., xcodebuild) in any way, ``use_xcode yes`` should be set or the port should be included in the xcode PortGroup.
```

Comment:
Ports cannot be included in PortGroups, ports include PortGroups (I know the term PortGroup suggests otherwise; it's not very-well named).

Reply:
Right.. that's weird. Done